examples/js/footer/footer-table.js

```diff
@@ -25,36 +24,34 @@ export default class FooterTable extends React.Component {
   }
 
   render() {
-    const options = {
-      footerData: [
-        [
-          {
-            label: 'Total',
-            target: 0
-          },
-          {
-            label: 'Total value',
-            target: 2,
-            align: 'right',
-            formatter: (tableData) => {
-              let label = 0;
-              for (let i = 0, tableDataLen = tableData.length; i < tableDataLen; i++) {
-                label += tableData[i].price;
-              }
-              return (
-                <strong>{ label }</strong>
-              );
+    const footerData = [
+      [
+        {
+          label: 'Total',
+          target: 0
+        },
+        {
+          label: 'Total value',
+          target: 2,
+          align: 'right',
+          formatter: (tableData) => {
+            let label = 0;
+            for (let i = 0, tableDataLen = tableData.length; i < tableDataLen; i++) {
+              label += tableData[i].price;
             }
+            return (
+              <strong>{ label }</strong>
+            );
           }
-        ]
-      ],
-      footerFormatterReturnData: products
-    };
+        }
+      ]
+    ];
+
     return (
       <div>
         <BootstrapTable
           data={ products }
-          options={ options }
+          footerData={ footerData }
```

Comment:
@AllenFang I've removed `footerData` from `options` and added as a prop to `BootstrapTable`.